gammapy/background/models.py

```diff
@@ -71,3 +81,775 @@ def evaluate(self, x, y):
         y = np.asanyarray(y, dtype=float)
         parvals = self.parvals(x)
         return self._evaluate_y(y, parvals)
+
+
+def _make_bin_edges_array(lo, hi):
+    """Make bin edges array from a low values and a high values array.
+
+    TODO: move this function to somewhere else? (i.e. utils?)
+
+    Parameters
+    ----------
+    lo : `~numpy.array`
+        lower boundaries
+    hi : `~numpy.array`
+        higher boundaries
+
+    Returns
+    -------
+    bin_edges : `~numpy.array`
+        array of bin edges as [[low], [high]]
+    """
+    return np.append(lo.flatten(), hi.flatten()[-1:])
+
+
+def make_linear_bin_edges_arrays_from_wcs(w, nbins_x, nbins_y):
+    """Make a 2D linear binning from a WCS object.
+
+    This method gives the correct answer only for linear X, Y binning.
+    The method expects angular quantities in the WCS object.
+    X is identified with WCS axis 1, Y is identified with WCS axis 2.
+    The method needs the number of bins as input, since it is not in
+    the WCS object.
+
+    TODO: move this function to somewhere else? (i.e. utils?)
+
+    Parameters
+    ----------
+    w : `~astropy.wcs.WCS`
+        WCS object describing the bin coordinates
+    nbins_x : `~int`
+        number of bins in X coordinate
+    nbins_y : `~int`
+        number of bins in Y coordinate
+
+    Returns
+    -------
+    bins_x : `~numpy.array` of `~astropy.coordinates.Angle`
```

Comment:
Just writing this is better:
```
bins_x : `~astropy.coordinates.Angle`
```
You can mention in the description that it's an array, which you already to in this case.